qiime2/core/testing/examples.py

```diff
@@ -264,3 +264,22 @@ def collection_dict_of_ints(use):
     )
 
     out.assert_output_type(semantic_type='SingleInt', key='Foo')
+
+
+def construct_and_access_collection(use):
+    ints_a = use.init_artifact('ints_a', single_int1_factory)
+    ints_b = use.init_artifact('ints_b', single_int2_factory)
+
+    rc_in = use.construct_collection(
+        'rc_in', 'artifact', {'a': ints_a, 'b': ints_b}
```

Comment:
Could we actually infer the type based on the var_types of the incoming dict/list?

Reply:
Is this resolved now by our discussion yesterday (dropping the `member_type` parameter)?